building/lambda/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN pip3 install -r /root/requirements.txt
 
 ADD requirements-dev.txt /root/
 # Removing "-e ." installation
-RUN head -n -2 /root/requirements-dev.txt > /root/temp.txt
```

Comment:
We should probably come up with a better way of doing this. Perhaps filtering out any line that starts with -e instead?